package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/squatch-js",
-  "version": "2.3.1",
+  "version": "2.3.2-23",
```

```suggestion
  "version": "2.3.2-23",
```
TODO: figure out what version we are releasing